core/client/app/templates/components/gh-ed-preview.hbs

```diff
@@ -1 +1,3 @@
-{{gh-format-markdown markdown}}
+{{#unless mediaQueries.maxWidth1000}}
```

Comment:
There's a problem here in that we have a toggle to switch between the editor and markdown when on mobile - as it currently stands that's broken with this PR.

I think we still want to render the preview and use CSS to show/hide it on mobile so that our scroll sync which is dependent on the height of the preview area still functions.